# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/go/hotfix-goarches.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- go/src/cmd/dist/build.go.vanilla	2023-11-30 14:05:06.291843468 +0100
+++ go/src/cmd/dist/build.go	2023-11-30 14:06:49.621850134 +0100
@@ -63,13 +63,18 @@
 var okgoarch = []string{
 	"386",
 	"amd64",
+	"amd64p32",
 	"arm",
+	"armbe",
 	"arm64",
 	"loong64",
 	"mips",
 	"mipsle",
 	"mips64",
+	"mips64p32",
 	"mips64le",
+	"mips64p32le",
+	"ppc",
 	"ppc64",
 	"ppc64le",
 	"riscv64",
